src/Kunstmaan/ApiBundle/Model/ApiPage.php

```diff
@@ -0,0 +1,129 @@
+<?php
+
+namespace Kunstmaan\ApiBundle\Model;
+
+
+use Doctrine\Common\Collections\ArrayCollection;
+use Kunstmaan\NodeBundle\Entity\Node;
+use Kunstmaan\NodeBundle\Entity\NodeTranslation;
+use Kunstmaan\NodeBundle\Entity\NodeVersion;
+use Kunstmaan\PagePartBundle\Helper\HasPagePartsInterface;
+
+class ApiPage
+{
+    /** @var string */
+    private $type;
+
+    /** @var HasPagePartsInterface */
+    private $page;
+
+    /** @var array|ArrayCollection */
+    private $pageParts;
+
+    /** @var Node */
+    private $node;
+
+    /** @var NodeTranslation */
+    private $nodeTranslation;
+
+    /** @var NodeVersion */
+    private $nodeVersion;
+
+    /**
+     * @return string
+     */
+    public function getType()
+    {
+        return $this->type;
+    }
+
+    /**
+     * @param string $type
+     */
+    public function setType($type)
+    {
+        $this->type = $type;
```

Comment:
Could the setters return the object itself? `return $this`

Reply:
done